File: uikit/src/main/java/io/agora/chat/uikit/chat/interfaces/OnMessageSendCallBack.java
package io.agora.chat.uikit.chat.interfaces;

import io.agora.chat.ChatMessage;

public interface OnMessageSendCallBack {
    /**
     * Callback after the message is sent successfully
     * @param message
     */
    default void onSuccess(ChatMessage message){}

    /**
     * Wrong message in chat
     * @param code
     * @param errorMsg
     */
    void onError(int code, String errorMsg);
}

File: ease-im-kit/src/main/kotlin/io/agora/uikit/feature/chat/interfaces/OnMessageSendCallback.kt
package io.agora.uikit.feature.chat.interfaces

import io.agora.uikit.common.ChatMessage

interface OnMessageSendCallback {
    /**
     * Callback after the message is sent successfully
     * @param message
     */
    fun onSuccess(message: ChatMessage?) {}

    /**
     * Wrong message in chat
     * @param code
     * @param errorMsg
     */
    fun onError(code: Int, errorMsg: String?)
}